packages/presentation/lib/l10n/intl_en.arb

```diff
@@ -17,5 +17,6 @@
   "loginIsRequired": "Login is required",
   "loginRegex": "Login field should be at least 8 symbols",
   "passwordIsRequired": "Password is required",
-  "passwordRegex": "Invalid password"
+  "passwordRegex": "Invalid password",
+  "emptyString": ""
```

Comment:
why do u need this value?

Reply:
there is a mandatory line in the bottom bar and navigation rail, but according to the layout we do not have an inscription, so as not to create constantly with an empty line, i created it here

Reply:
let's create an extension for emptyString. it doesn't relate to localization